### ISSUE TITLE
Talk to text Accessibility 

### ISSUE THREAD

**Issue by augusteemond** [sagittMZ/qa_guru#37]
Virtual writing formats have many tools that regard to accessibility. One feature I believe should be more accessible on the toolbar itself is the talk to text feature. This can be incorporated fairly easy and would be used very frequently when brainstorming an Idea.
